src/iuse.cpp

```diff
@@ -7895,17 +7895,14 @@ int iuse::multicooker(player *p, item *it, bool t, const tripoint &pos)
         }
 
         if (cooktime <= 0) {
-            it->active = false;
-
-            item meal(it->get_var( "DISH" ), calendar::turn);
-            meal.active = true;
-
-            if (meal.has_flag("EATEN_HOT")) {
-                meal.item_tags.insert("HOT");
+            item& meal = it->emplace_in( it->get_var( "DISH" ) );
+            if( meal.has_flag( "EATEN_HOT" ) ) {
+                meal.active = true;
+                meal.item_tags.insert( "HOT" );
                 meal.item_counter = 600;
             }
 
-            it->put_in(meal);
```

Comment:
Previously modifying `meal` after `put_in` would result in the local and contained copy differing